dist/tools/cppcheck/check.sh

```diff
@@ -8,7 +8,10 @@
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 
-: "${RIOTTOOLS:=${PWD}/dist/tools}"
+: "${RIOTBASE:=$(cd $(dirname $0)/../../../; pwd)}"
+cd $RIOTBASE
```

Comment:
Why do you need to change the current directory to RIOTBASE ?

Reply:
because right now the script assumes to be run from RIOTBASE from the start. As per title, this PR allows the script to be called from anywhere, but that makes it necessary to change to RIOTBASE if not already run from there.